### DISCUSSION TITLE
useModel报ts类型错误如何解决

### DISCUSSION THREAD

**Discussion by cjx-dev** [umijs/umi#7629] [Q&A]
![image](https://user-images.githubusercontent.com/64702233/143674063-21e4d845-a0c4-4eac-b859-25ca59cc0551.png)

**Comment by nonevx** [comment]
同问, plugin-initial-state插件启动以后, plugin-model原来的使用方法就无效了么....
![image](https://user-images.githubusercontent.com/25873233/167628507-66109de1-c09b-4a12-9e60-8daf7d1abf1e.png)

**Reply by leftstick** [reply]
能提供个可复现的repo么？

**Comment by zozopen** [comment]
请问解决了吗
